src/components/Text.tsx

```diff
@@ -21,7 +21,7 @@ type TextProps = RNTextProps &
         children: React.ReactNode;
 
         /** The family of the font to use */
-        family?: keyof typeof fontFamily;
+        family?: keyof FontUtilsType['fontFamily']['platform'];
```

Comment:
```suggestion
        family?: keyof FontUtilsType.fontFamily.platform;
```

or is it a weird typescript thing

Reply:
I don't think this is possible with TS

<img width="1223" alt="Screenshot 2023-12-16 at 14 41 40" src="https://github.com/Expensify/App/assets/20173411/cf484bff-fa4a-4ab8-a1f2-de5d11c9053d">

Reply:
Ok cool. Why they did this... seems weird haha